src/Runner.Worker/action_yaml.json

```diff
@@ -133,7 +137,7 @@
                     "working-directory": "string-steps-context",
                     "shell": {
                         "type": "string-steps-context",
-                        "required": true
+                        "required": false
```

Comment:
we should still fail here if composite defaults aren't set and a shell isn't set for that step. Do we currently do that?

Reply:
Yes, in that case, the action would fail with this output:
![image](https://user-images.githubusercontent.com/97525037/184882792-350f3e4e-8410-456a-a4dc-7f726f6d49a5.png)